src/components/pages/entities/EntityPreviewFileCard.vue

```diff
@@ -0,0 +1,189 @@
+<template>
+  <div class="preview-card flexcolumn" :key="previewFile.id">
+    <entity-preview
+      :entity="{
+        preview_file_id: previewFile.id,
+        preview_file_extension: previewFile.extension
+      }"
+      :empty-height="200"
+      :empty-width="300"
+      :height="200"
+      :width="300"
+      is-rounded-top-border
+      show-movie
+    />
+    <div class="preview-card-data">
+      <div class="flexrow">
+        <span class="card-revision tag strong flexrow-item">
+          v{{ previewFile.revision }}
+        </span>
+        <span
+          class="card-file-name flexrow-item"
+          :title="previewFile.original_name"
+        >
+          {{ `${previewFile.original_name}` }}
+        </span>
+        <span class="filler"></span>
+        <span class="card-extension flexrow-item mr0">
+          {{ previewFile.extension }}
+        </span>
+      </div>
+      <div class="flexrow mt1">
+        <span
+          class="preview-status mr05"
+          :title="previewFile.validation_status"
+          :style="getPreviewValidationStyle(previewFile)"
+        >
+          &nbsp;
+        </span>
+        <people-avatar
+          class="person"
+          :person="personMap.get(previewFile.person_id)"
+          :font-size="10"
+          :size="20"
+        />
+        <span class="filler"></span>
+        <span
+          class="card-file-size flexrow-item mr05"
+          v-if="previewFile.file_size"
+        >
+          {{ renderFileSize(previewFile.file_size) }}B
+        </span>
+        <a
+          class="button"
+          :href="getDownloadPath(previewFile.id)"
+          :title="$t('playlists.actions.download_file')"
+          v-if="!isCurrentUserArtist"
+        >
+          <download-icon class="icon is-small" />
+        </a>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex'
+import { DownloadIcon } from 'lucide-vue-next'
+
+import { renderFileSize } from '@/lib/render'
+
+import EntityPreview from '@/components/widgets/EntityPreview.vue'
+import PeopleAvatar from '@/components/widgets/PeopleAvatar.vue'
+
+export default {
+  name: 'entity-preview-files',
```

Comment:
wrong component name

```suggestion
  name: 'entity-preview-file-card',
```